docs/dagster-university/pages/dagster-essentials/lesson-3/coding-practice-taxi-zones-file-asset.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Lesson 3: Practice: Create a taxi_zones_file asset"
+module: "dagster_essentials"
+lesson: "3"
+---
+
+# Practice: Create a taxi_zones_file asset
+
+To practice what you’ve learned, create an asset in `trips.py` that:
+
+- Is named `taxi_zones_file`. This asset will contain a unique identifier and name for each part of NYC as a distinct taxi zone.
+- Uses the `requests` library to obtain [NYC Taxi Zones](https://data.cityofnewyork.us/Transportation/NYC-Taxi-Zones/d3c5-ddgc) data from [this link](https://data.cityofnewyork.us/api/views/755u-8jsi/rows.csv?accessType=DOWNLOAD) ([`https://data.cityofnewyork.us/api/views/755u-8jsi/rows.csv?accessType=DOWNLOAD`](https://data.cityofnewyork.us/api/views/755u-8jsi/rows.csv?accessType=DOWNLOAD))
```

Comment:
This link doesn't render properly in the preview

Reply:
I'll take a look, ty